index.html

```diff
@@ -0,0 +1,193 @@
+      <ul class="skils">
+       <li class="button">html</li>
+       <li class="skil button">bootstrap</li>
+       <li class="skil button">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li class="button">html</li>
+       <li class="skil button">bootstrap</li>
+       <li class="skil button">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    
+</section>
+
+<section class="section-3">
+   <div class="about-me">
+      <h1 class="h31">About me</h1>
+      <p class="p31">Hello I’m a software developer! I can help you build a product ,feature or 
+         website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <a class="get-myresume button" href="">Get My Resume</a>
+      <img class="abt-me-image" src="./assets/images/Group1.png" alt="">
+   </div>
+   <hr class="line-3">
+
+   <ul class="lists">
+      <li class="Lan">Language 
+          <ul class="lan-list">
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>Css</li>
+            </ul>
+      </li>
+        
+      <li class="Frame"> Framework
+          <ul class="Frame-list">
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+          </ul>
+      </li>
+      <li class="Sk">Skills
+          <ul class="Sk-list">
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+      </li>
+  </ul>
+</section>
+
+<form class="contact-form" action="https://formspree.io/f/{Portifolio-contact}" method="post">
+  <ul class="ul-contact">
+    <li class="p-contact ">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</li>
+    <li>
+      <label for="name"></label>
+      <input type="text" id="name" name="user_name" placeholder="Full name" maxlength="30" required>
+    </li>
+    <li>
+      <label for="mail"></label>
+      <input type="email" id="mail" name="user_email" placeholder="Email address" required>
+    </li>
+    <li>
+      <label for="msg"></label>
+      <textarea id="msg" name="user_message" placeholder="Enter text here" maxlength="500" required></textarea>
+    </li>
+    <li class="submit-button">
+      <button type="submit" id="get-in-touch">Get in touch</button>
+    </li>
```

Comment:
Good job trying to implement the button interactivity. However, the implementation is not correctly done because when I press the button, the pressed state is green which is not consistent with the Figma template and also I do not see the hover state yet implemented. Kindly add these features correctly in order to provide a good UX for users when they are viewing your portfolio.

Furthermore, there is this white border around the button, I think removing this will be a good idea and will make the UI much cleaner.

YOUR DESIGN

FIGMA TEMPLATE

[screen-recorder-wed-jan-25-2023-01-51-04.webm](https://user-images.githubusercontent.com/84930355/214455145-a8f84279-834f-4def-bc3d-fa3a6f3f7ed5.webm)


<img width="323" alt="hovers" src="https://user-images.githubusercontent.com/84930355/214447940-20d47f4e-d800-40ee-905b-166b8efe3a22.png">